src/styles/index.scss

```diff
@@ -1,3 +1,92 @@
-body {
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    margin-top: 33px;
+  }
+
+  &__model {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    width: 166px;
+    height: 36px;
+    margin-top: 40px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    align-self: start;
+    width: 98px;
+    height: 14px;
+    padding-left: 17px;
+    padding-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__grade {
+    display: flex;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-stars {
+      background-image: url(../images/stars.png);
+      width: 96px;
+      height: 16px;
+    }
+
+    &-reviews {
+      align-self: center;
+      width: 53px;
+      height: 16px;
+      line-height: 16px;
+      font-size: 10px;
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    background-image: url(../images/price.png);
```

Comment:
Using images for elements like price seems to be a non-semantic approach. If it's a static component, consider using HTML and CSS to create the price's appearance instead of an image.